Dfe.Data.SearchPrototype/Web/Views/Home/Index.cshtml

```diff
@@ -68,10 +68,8 @@
                     @foreach (var searchItem in Model.SearchItems!)
                     {
                         <li id="name-@searchItem.Urn">
-                            <h4>
-                                <a href="/">
-                                    @searchItem.Name
-                                </a>
+                            <h4 style="background-color:aliceblue">
```

Comment:
It is a lovely blue - probably not GDS compliant but I guess we may as well have fun while we're allowed to